src/common/types/objects/ledger_entries.ts

```diff
@@ -1,5 +1,47 @@
 import {SignerEntry} from './index'
 
+export interface AccountRootLedgerEntry {
+  LedgerEntryType: 'AccountRoot',
+  Account: string,
+  Flags: number,
+  Sequence: number,
+  Balance: string,
+  OwnerCount: number,
+  PreviousTxnID: string,
+  PreviousTxnLgrSeq: number,
+  AccountTxnID?: string,
+  RegularKey?: string,
+  EmailHash?: string,
+  MessageKey?: string
+  TickSize?: number,
+  TransferRate?: number,
+  Domain?: string
+}
```

Comment:
A quick glance at the code tells me it may still be possible to get (and set!) `WalletLocator` and `WalletSize` fields with the AccountSet transaction. They've been marked deprecated for a long time, but still, may be better to have them here so you don't error out if you encounter one?